### PR TITLE
Implement updated Psammead module to remove the bottom border beneath the top nav

### LINKED ISSUE #6602: Remove top navigation bottom border
**NB - this issue exists within Psammead, not Simorgh**

Simorgh PR - https://github.com/bbc/simorgh/pull/7305
Psammead PR - https://github.com/bbc/psammead/pull/3628 ✅ 
Version bump PR - https://github.com/bbc/psammead/pull/3629 ✅ 

**Describe the bug**
There is a 1px [pink line](https://user-images.githubusercontent.com/34196381/82549219-1c218500-9b4c-11ea-9635-75fdd0c9dcde.png)  below the seconday nav bar that needs to be removed. It wasn’t intended design but a technical limitation - the way the component was built to allow the items in the nav wrapping over two lines meant that the line couldn’t stretch over 1280px.

We also have an [additonal px](https://user-images.githubusercontent.com/34196381/82549248-2cd1fb00-9b4c-11ea-872e-dff3960befb2.png) of padding below the hover state of the News nav bar.

This impacts all page types.

This issue might be related to this https://github.com/bbc/simorgh/issues/5864

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to any Simorgh page
2. See issues mentioned above.

**Expected behaviour**
There should be no padding between the hover state on the News nav bar and the pink line below the secondary nav should be removed.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [x] This bug fix is expected to need manual testing.

**Additional context**
If picked up by ws-stories or ws-media speak to ws-home for more context as they built this component.
Denis Hernandez worked on the nav so may be good to consult with him when picking this up.

### PR DESCRIPTION
**Pull request by RichardPK**
Resolves #6602

**Overall change:**
- Implement `psammead-navigation: "^6.0.15"` which obscures the line bellow the top nav while maintaining it when the nav breaks across two rows. Psammead PR here: https://github.com/bbc/psammead/pull/3628
- Bumping a lot of packages because of a dependency chain that revolves around the `gel-foundations: 4.2.0` update
- Updated snaps

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have assigned this PR to the Simorgh project

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -14645,7 +14652,7 @@
     },
     "jest-silent-reporter": {
       "version": "0.2.1",
-      "resolved": "https://registry.npmjs.org/jest-silent-reporter/-/jest-silent-reporter-0.2.1.tgz",
+      "resolved": "https://npm.morph.int.tools.bbc.co.uk/jest-silent-reporter/-/jest-silent-reporter-0.2.1.tgz",
```
Comment (ryanmccombe) [inline package-lock.json]: got morph registry stuff in here ☹️

**Review by ryanmccombe — APPROVED** [review]

**Review by karinathomasbbc — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (RichardPK) [reply package-lock.json]: lol belter

Comment (RichardPK) [reply package-lock.json]: Thank you for spotting. I think my npm alias attempts to change this relentlessly

File: src/app/containers/Include/__snapshots__/index.test.jsx.snap [hunk 2]
```diff
@@ -19,11 +19,13 @@ exports[`IncludeContainer should render a fallback for VJs on an Amp page when i
       class="StyledEmbedError-sc-1oc1egd-0 fyPX"
     >
       <div
-        class="StyledErrorMessage-sc-1oc1egd-1 eCvEfh"
+        class="StyledErrorMessage-sc-1oc1egd-1 jAchtl"
       >
-        <strong>
-          Sorry, we can’t display this part of the story on this lightweight mobile page.
-        </strong>
+        <div>
```
Comment (RichardPK) [inline src/app/containers/Include/__snapshots__/index.test.jsx.snap:24]: Updating the embed-error component to the latest version has changed this snapshot, https://github.com/bbc/psammead/blob/156815c128f560819c1bacee82bda71e4522a06c/packages/components/psammead-embed-error/src/index.jsx#L71

**Pull request merged**